Yoast/wordpress-seo
# Added a class for the checkmark.

## Summary

This PR can be summarized in the following changelog entry:

## Relevant technical choices:

* Added a green checkmark, needed in premium. It might be a handy one for the licenses page.

## Test instructions

This PR can be tested by following these steps:

* Do `grunt build:css`
* Check if everything still works

Needed for https://github.com/Yoast/wordpress-seo-premium/issues/1048
